src/components/Flashlight.js

```diff
@@ -7,7 +7,8 @@ import './style/Flashlight.css';
 
 class Flashlight extends React.Component {
 
-    showFlashlight = () => {
+    showFlashlight = e => {
+        e.preventDefault();
         const spotlight = document.getElementById("spotlight");
```

Comment:
Avoid using direct DOM manipulation when possible, though I know the flashlight component is a bit tricky. Not merge-blocking for now, but something we should refactor!